src/Collection/Collection.php

```diff
@@ -50,6 +51,28 @@ public function __construct($items)
     }
 
     /**
+     * Returns a string representation of this object that can be used
+     * to reconstruct it
+     *
+     * @return string
+     */
+    public function serialize()
+    {
+        return serialize($this->buffered());
+    }
+
+    /**
+     * Unserializes the passed string and rebuilds the Collection instance
+     *
+     * @param string $collection The serialized collection
+     * @return void
+     */
+    public function unserialize($collection)
```

Comment:
Without reading the docblock my first intuition is `$collection` suggests it a collection not string. But okay.